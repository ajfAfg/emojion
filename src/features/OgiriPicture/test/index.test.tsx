import { render } from "@testing-library/react";
import { OgiriPicture } from "..";

describe("OgiriPicture", () => {
  it("renders the ÅŒgiri picture unchanged", async () => {
    const { container } = render(<OgiriPicture emoji="ðŸ¥³" text="dummy text" />);
    expect(container).toMatchSnapshot();
  });
});
