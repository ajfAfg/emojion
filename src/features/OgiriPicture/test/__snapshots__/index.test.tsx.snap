// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OgiriPicture renders the ÅŒgiri picture unchanged 1`] = `
<div>
  <div
    tw="flex bg-white flex-col justify-center items-center"
  >
    <div
      tw="w-60 h-60 mx-auto mt-12 flex justify-center items-center text-[15rem]"
    >
      ðŸ¥³
    </div>
    <div
      tw="flex max-w-full"
    >
      <div
        tw="shrink px-4 py-2 min-h-[3rem] border-[#A6ADBA] rounded-lg flex flex-col mx-32 my-8 w-full"
      >
        <span
          style="word-break: break-all;"
          tw="w-full text-[#191D24] text-center justify-center bg-transparent text-5xl"
        >
          dummy text
        </span>
      </div>
    </div>
  </div>
</div>
`;
